flexmeasures/data/schemas/tests/test_sensor.py

```diff
@@ -6,32 +6,36 @@
 
 
 @pytest.mark.parametrize(
-    "sensor_id, src_quantity, dst_unit, fails",
+    "sensor_id, src_quantity, dst_unit, fails, exp_dst_quantity",
     [
         # deserialize a sensor
-        (1, None, "MWh", False),
-        (1, None, "kWh", False),
-        (1, None, "kW", False),
-        (1, None, "EUR", True),
-        (2, None, "EUR/kWh", False),
-        (2, None, "EUR", True),
+        (1, None, "MWh", False, None),
+        (1, None, "kWh", False, None),
+        (1, None, "kW", False, None),
+        (1, None, "EUR", True, None),
+        (2, None, "EUR/kWh", False, None),
+        (2, None, "EUR", True, None),
         # deserialize a quantity
-        (None, "1MWh", "MWh", False),
-        (None, "1 MWh", "kWh", False),
-        (None, "1 MWh", "kW", True),
-        (None, "100 EUR/MWh", "EUR/kWh", False),
-        (None, "100 EUR/MWh", "EUR", True),
+        (None, "1MWh", "MWh", False, "1 MWh"),
+        (None, "1 MWh", "kWh", False, "1000.0 kWh"),
+        (None, "1 MWh", "kW", True, None),
+        (None, "100 EUR/MWh", "EUR/kWh", False, "0.1 EUR/kWh"),
+        (None, "100 EUR/MWh", "EUR", True, None),
+        (None, "1 EUR/kWh", "/MWh", False, "1000.0 EUR/MWh"),
```

Comment:
I would add some test cases where the both the numerator and denominator are empty: `/`.  I guess this should be tested in `src_quantity` and `dst_unit`.

Reply:
Good suggestions:
1. I'm adding a test case for `src_quantity="/"`.
2. `dst_unit="/"` translates to `to_unit="/"`, which should already fail initialization before failing deserialization, so I don't consider this a deserialization test case. However, I did add logic to catch this case and throw a better error for developers.